src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectDifference.cs

```diff
@@ -8,10 +8,9 @@ namespace Microsoft.CodeAnalysis.Razor.ProjectSystem;
 [Flags]
 internal enum ProjectDifference
 {
-    None = 0,
-    ConfigurationChanged = 1,
-    ProjectWorkspaceStateChanged = 2,
-    DocumentAdded = 4,
-    DocumentRemoved = 8,
-    DocumentChanged = 16,
+    None = 1 << 0,
+    ConfigurationChanged = 1 << 1,
+    DocumentAdded = 1 << 2,
+    DocumentRemoved = 1 << 3,
+    DocumentChanged = 1 << 4,
```

Comment:
This produces different values. `None` should always be 0, but `1 << 0` is 1.

Reply:
Yep, but the whole file got deleted in the end :)

Reply:
Yeah, that's the danger of going commit-by-commit sometimes. :smile: